documentation/Installation.md

```diff
@@ -4053,15 +4058,15 @@ rbac:
 * The custom policies should not have 'oob-' prefix.
 * To manage custom policies on an existing cluster use the `manage_psp` maintenance procedure. 
 
-### Admission pss
+### Admission PSS
```

Comment:
Hi @disa1217,
Thank you for doc review. I suppose it was used _pss_ here to refer to the really used case-sensitive keyword. See examples of the configuration below:
```yaml
rbac:
  admission: pss
  pss:
    ...
```
I am not against changing to _PSS_ in this particular place, but I think [Admission psp](https://github.com/Netcracker/KubeMarine/blob/v0.29.0/documentation/Installation.md?plain=1#L3728) as much as [references](https://github.com/Netcracker/KubeMarine/blob/v0.29.0/documentation/Installation.md?plain=1#L64) to the sections in the title should be aligned.

Reply:
hi @ilia1243 ,
You can reject this change.
Or, should I update it?

Reply:
Please revert this change, or align "Admission psp" and references to these section in the title.

Reply:
Reverted.